maintainers/maintainer-list.nix

```diff
@@ -8200,6 +8200,11 @@
     githubId = 2405974;
     name = "Sébastian Méric de Bellefon";
   };
+  hellodword = {
+    github = "hellodword";
+    githubId = 46193371;
+    name = "hellodword";
+  };
```

Comment:
Please split the maintainer addition to its own commit, `maintainers: add hellodword`